libs/libc/netdb/Kconfig

```diff
@@ -237,6 +237,11 @@ config NETDB_DNSSERVER_IPv4ADDR
 		Default DNS server IPv4 address in host byte order.  Default value
 		10.0.0.1.  This may be changed via dns_add_nameserver().
 
+config NETDB_DNS_STREAM
+	bool "DNS supports TCP transmission"
+	depends on LIBC_NETDB
+	default n
```

Comment:
Please include a --help-- to explain exactly what is this feature, when it should be used, etc

Reply:
Done